hack/check-latest-images.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# Copyright The Shipwright Contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# Check the latest image and replace
+
+# Usage
+# ./check-latest-image.sh <FILE> <IMAGE> <IMAGE_URL>
+
+set -euo pipefail
+
+# Check the number of arguments passed and exit if less than three
+if [ $# -lt 3 ]; then
+	echo "USAGE: $0 <FILE> <IMAGE> <IMAGE_URL>"
+	exit 1
+fi
+
+# Set environment variables
+IMAGE=$(printf '%s\n' "$(echo ${1})" | sed -e 's/[]\/$*.^[]/\\&/g')
```

Comment:
A comment would be helpful to understand what you are doing here.

Reply:
Escape image URL char with a backslash. I will add the comment.